tool/README.md

```diff
@@ -24,29 +24,28 @@ Make sure:
 
 ### Prepare the release
 
-#### Update the DevTools version number
-
-- Make sure your working branch is clean
-- Run the `tool/release_helper.sh` script with `minor` or `major`.
-   `./tool/release_helper.sh [minor|major]`
-- This creates 2 branches for you:
-    - Release Branch
-    - Next Branch
-- The following steps will guide you through how these branches will be prepared and merged.
-- **For your convenience, the `tool/release_helper.sh` script exports the following two variables to the terminal it is run in:**
-  - `$DEVTOOLS_RELEASE_BRANCH`
-  - `$DEVTOOLS_NEXT_BRANCH`
-
-#### Verify the version changes for `$DEVTOOLS_RELEASE_BRANCH`
-
-Verify release_helper.sh script:
+#### Ensure you have Github CLI installed
+- If you need to install the Github CLI you can run:
+    `brew install gh`
+- More information can be found at: https://cli.github.com/manual/installation
+
+#### Create a release PR
+- Run: `./tool/release_helper.sh`
+- This will create a PR for you using the tip of master.
+- The branch for that PR will be checked out locally for you.
+- It will also update your local version of flutter to the Latest flutter candidate
```

Comment:
will the script fail if a person's flutter branch has local changes?

Reply:
Looking at our helper script it looks like it forcefully stashes those changes for the user before switching